src/lib/types/Message.ts

```diff
@@ -1,7 +1,9 @@
-export interface Message {
+import type { Timestamps } from "./Timestamps";
+
+export type Message = Partial<Timestamps> & {
```

Comment:
We should use the whole `Timestamps` imo.

If you only want `createdAt` or `updatedAt`, you can use `Pick`

Reply:
up to you :)